features/Jars/JarCollapsible.tsx

```diff
@@ -311,13 +312,14 @@ export const JarCollapsible: FC<{
               <Tooltip
                 text={
                   isYearnJar
-                    ? `This jar deposits into Yearn's ${APYs[1].vault
-                    }, The base rate of ${apr.toFixed(
-                      2,
-                    )}% is provided by the underlying Yearn strategy`
+                    ? `This jar deposits into Yearn's ${
+                        APYs[1]?.vault
```

Comment:
This is the relevant change. Depending on the order of requests, APYs might be empty when reloaded.